webapp/src/store/tasks.ts

```diff
@@ -22,12 +27,18 @@ export const useTasksStore = defineStore('tasks', () => {
 
   async function initTasks (): Promise<void> {
     try {
-      const tasks = await fetchTasks(CONFIG.serverUrl)
+      const tasks = (await fetchTasks(CONFIG.serverUrl)).filter((t: Task) => !TASKS_TO_FILTER_OUT.includes(t.id))
       tasks.forEach(addTask)
+      loadingAlreadyFailed.value = false
     } catch (e) {
       console.error('Fetching of tasks failed with error', e instanceof Error ? e.message : e)
-      const toaster = useToaster()
-      toaster.error('The server is unreachable. \n Please try again later or reach out on slack.')
+
+      //Only display UI message once
+      if (loadingAlreadyFailed.value === false) {
```

Comment:
always def and boolean

```suggestion
      if (!loadingAlreadyFailed.value) {
```